apps/studio/pages/project/[ref]/storage/vectors/buckets/[bucketId].tsx

```diff
@@ -21,15 +25,30 @@ const VectorsBucketPage: NextPageWithLayout = () => {
   } = useVectorBucketQuery({ projectRef, vectorBucketName: bucketId })
 
   return (
-    <div className="storage-container flex flex-grow p-4">
+    <div className="storage-container flex flex-grow">
       {isError && <StorageBucketsError error={error as any} />}
 
-      {isLoading && <GenericSkeletonLoader className="w-full" />}
+      {isLoading && (
+        <ScaffoldContainer>
+          <ScaffoldSection isFullWidth>
+            <GenericSkeletonLoader />
+          </ScaffoldSection>
+        </ScaffoldContainer>
+      )}
```

Comment:
Just nesting this loader in basic scaffolding to make the transition a bit nicer.